robot/Cumulus/resources/NPSPSettingsPageObject.py

```diff
@@ -49,6 +49,15 @@ def open_sub_link(self,title):
                                                        error=f"click on {title} sublink was not successful even after 30 seconds")
         self.selenium.capture_page_screenshot()
 
+    @capture_screenshot_on_error
+    def verify_sub_link_present(self, title, bool):
+        """ Asserts if the sublink is present or not present
+            eg: Verify Sub Link Present     <Linkoption>   True/False
+        """
+        locator=npsp_lex_locators['npsp_settings']['panel_sub_link'].format(title)
+        assert bool == str(self.npsp.check_if_element_exists(locator)), "Expected value to be {} but found {}".format(
+            bool, self.npsp.check_if_element_exists(locator)
```

Comment:
curious on why are you trying to print the keyword used as the error message when this fails

Reply:
Updated the message to be meaningful.